src/Workspaces/Remote/ServiceHub/Services/FindUsages/RemoteFindUsagesService.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes.FullyQualify;
```

Comment:
💭 Find Usages is using the Fully Qualify namespace?

Reply:
oopsie. :)